app/services/file_mover.py

```
@@ -29,14 +29,18 @@ def resilient_unlink(path: Path) -> None:
         path.unlink()
 
 
-def resilient_copy(source: Path, destination: Path, progress_callback: Optional[Callable[[int], None]] = None) -> None:
+def resilient_copy(
+    source: Path, destination: Path, progress_callback: Optional[Callable[[int], None]] = None
+) -> None:
     """Resiliently copy a file with retries, exponential backoff, and clean destination recreation."""
+
     @retry_with_backoff(max_attempts=3, allowed_exceptions=(OSError,))
     def _do_copy():
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_do_copy`

Add return type annotation: `None`

(ANN202)